src/options.js

```diff
@@ -5,7 +5,20 @@ import {color, descending, quantile} from "d3";
 const TypedArray = Object.getPrototypeOf(Uint8Array);
 const objectToString = Object.prototype.toString;
 
-// This allows transforms to behave equivalently to channels.
+/**
+ * Given an iterable *data* and some *value* accessor, returns an array (a column) of the specified *arrayType* with the corresponding value of each element of the data. The *value* accessor may be one of the following types:
+ *
+ * * a string - corresponding to the field accessor (`d => d[value]`)
+ * * an accessor function - called as *arrayType*.from(*data*, *value*)
+ * * a number, Date, or boolean — resulting in an array uniformly filled with the *value*
+ * * an object with a transform method — called as *value*.transform(*data*)
+ * * an array of values - returning the same
+ * * null or undefined - returning the same
+ *
+ * If *arrayType* is specified, it must be Array or a similar class that implements the [Array.from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) interface such as a typed array. When *type* is Array or a typed array class, the return value of valueof will be an instance of the same (or null or undefined). If *type* is not specified, valueof may return either an array or a typed array (or null or undefined).
```

Comment:
Here we have one *type* that should be *arrayType*, but I feel we should do the opposite, and rename the argument to *type* instead of *arrayType*?

Reply:
done!